.github/workflows/instascale-release.yml

```diff
@@ -13,10 +13,14 @@ on:
         description: 'Published version of multi-cluster-app-dispatcher to be used as a dependency for InstaScale'
         required: true
         default: 'v0.0.0-dev'
-      is-latest:
-        description: 'Select if the built image should be tagged as latest'
+      is-stable:
```

Comment:
Why did you change this from `latest` to `stable`

Reply:
It matches what our release ADR proposed. On more detailed reviewing however... it looks like `dev` is actually the tag that should be updated. We should probably only update the `stable` tag as part of one of the final steps in the codeflare-operator release process. Does that make sense? https://github.com/project-codeflare/adr/blob/main/PCF-ADR-0003-codeflare-release-process.md#images

Reply:
@anishasthana This workflow is triggered for a release, so it should use `stable` tag.
`dev` should be updated in a separate workflow which would be triggered after a push event (workflow doesn't exist yet).

Reply:
That makes sense.